scripts/uninstall_stairwell_windows.ps1

```diff
@@ -28,45 +42,84 @@ if($Inception -eq $True) {
     
 }
 
-# Test for forwarder versions 1.4.0 and 1.4.1
-$PgmDir = Test-Path "C:\Program Files\Stairwell"
-if($PgmDir -eq $True) {
-    if($Null -eq $MaintToken) {
-        Write-Output "This requires a forwarder maintenance token from your environment"
-        Write-Output "Visit https://docs.stairwell.com/docs/find-a-maintenance-token for details"
-        $MaintToken = Read-Host -Prompt "Please enter your Stairwell Maintenance Token"
+# Test for specific 1.4.x versions, these require several specific things to uninstall successfully.
+# First we need to determine which specific version is installed then download the installer bundle.
+# Uninstallation w/the installer package is highly recommended, lots of problems otherwise.
+
+# Attempt to find our installed version of the forwarder
+$Package = Get-package "Stairwell Forwarder"
```

Comment:
I'm not terribly familiar with Powershell but I'd be tempted to extract the version into a variable:

```
$Package = Get-package "Stairwell Forwarder"
$WmiObj = Get-WmiObject -Class Win32_Product -Filter "Name = 'Stairwell Forwarder'"

$Version = ""
if ($Package.version) {
  $Version = $Package.version
} elseif ($WmiObj.version) {
  $Version = $WmiObj.version
}

if ("1.4.0.886","1.4.1.896" -contains $Version) {
...
```